src/configs/algo_config.py

```diff
@@ -131,4 +131,23 @@
     "exp_keys": ["similarity_metric", "selection_strategy", "consensus"]
 }
 
```

Comment:
@nikitaparate for running a new algo, you just want to do something similar here where you port over the object from non iid config to algo_config and sys_config